Tasks/AzureCLI/azureclitask.ts

```diff
@@ -198,9 +199,9 @@ export class azureclitask {
         }
     }
 
-    private static createPublishSettingFile(subscriptionName:string, subscriptionId:string, certificate:string, publishSettingFileName:string): void  {
+    private static createPublishSettingFile(subscriptionName:string, subscriptionId:string, certificate:string, serviceManagementUrl:string, publishSettingFileName:string): void  {
         //writing the data to the publishsetting file
```

Comment:
Just verify if _serviceManagementUrl_ can have a terminal slash and if so, will it cause any issues.